sdk/ml/azure-ai-ml/tests/pipeline_job/unittests/test_pipeline_job_schema.py

```diff
@@ -1729,6 +1729,7 @@ def test_dump_pipeline_inputs(self):
             "integer_input": 15,
             "bool_input": False,
             "string_input": "hello",
+            "string_integer_input": "43",
```

Comment:
why didn't the yaml file change included in PR

Reply:
those inputs are not existed on pipeline job yaml. Instead they are set by param_override